source/synthDrivers/oneCore.py

```diff
@@ -81,6 +84,13 @@ def convertCharacterModeCommand(self, command):
 		# Therefore, we don't use it.
 		return None
 
+	def convertLangChangeCommand(self, command):
+		lcid = languageHandler.localeNameToWindowsLCID(command.lang)
+		if lcid in (0, languageHandler.LOCALE_CUSTOM_UNSPECIFIED):

```

Comment:
How do you get a value of zero and what does it represent? Is there a test for this case?